File: ympa_fsharp_giraffe_server/src/model/OrderLabelDTO.fs
namespace ympa_fsharp_giraffe_server.Model

open System
open System.Collections.Generic
open ympa_fsharp_giraffe_server.Model.ParcelBoxLabelDTO

module OrderLabelDTO =

  //#region OrderLabelDTO


  type OrderLabelDTO = {
    OrderId : int64;
    PlacesNumber : int;
    Url : string;
    ParcelBoxLabels : ParcelBoxLabelDTO[];
  }
  //#endregion

File: src/main/kotlin/ympa_kotlin_wiremock_server/models/OrderLabelDTO.kt
@file:Suppress(
    "RemoveRedundantQualifierName",
    "unused",
)

package ympa_kotlin_wiremock_server.models

import com.fasterxml.jackson.annotation.JsonProperty

data class OrderLabelDTO(
    @field:JsonProperty("orderId")
    val orderId: kotlin.Long,

    @field:JsonProperty("placesNumber")
    val placesNumber: kotlin.Int,

    @field:JsonProperty("url")
    val url: kotlin.String,

    @field:JsonProperty("parcelBoxLabels")
    val parcelBoxLabels: kotlin.collections.List<ParcelBoxLabelDTO>,

)
